pkg/controllers/resources/pods/translate/hostpath.go

```diff
@@ -25,6 +25,9 @@ const (
 func (t *translator) ensureMountPropagation(pPod *corev1.Pod) {
 	for i, container := range pPod.Spec.Containers {
 		for j, volumeMount := range container.VolumeMounts {
+			// handle scenarios where path ends with a /
+			volumeMount.MountPath = strings.TrimSuffix(volumeMount.MountPath, "/")
```

Comment:
Could you add unit test case for it to the `TestVolumeTranslation`?

Reply:
I'll merge this for now, we can do a separate PR with the unit test